ts/features/euCovidCert/screens/BaseEuCovidCertificateLayout.tsx

```diff
@@ -1,50 +1,30 @@
-import { View } from "native-base";
 import * as React from "react";
 import { useRef } from "react";
-import { Image, SafeAreaView, ScrollView, StyleSheet } from "react-native";
+import { SafeAreaView, ScrollView, StyleSheet } from "react-native";
 import { NavigationEvents } from "react-navigation";
-import image from "../../../../img/features/euCovidCert/eu-flag.png";
-import { H1 } from "../../../components/core/typography/H1";
-import { H2 } from "../../../components/core/typography/H2";
+import { View } from "native-base";
+import { heightPercentageToDP } from "react-native-responsive-screen";
 import { IOStyles } from "../../../components/core/variables/IOStyles";
 import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
-import I18n from "../../../i18n";
 import { WithTestID } from "../../../types/WithTestID";
 import { setAccessibilityFocus } from "../../../utils/accessibility";
 import { emptyContextualHelp } from "../../../utils/emptyContextualHelp";
 import SectionStatusComponent from "../../../components/SectionStatus";
 
 type Props = WithTestID<{
+  header?: React.ReactElement;
   content: React.ReactElement;
   footer?: React.ReactElement;
 }>;
 
 const styles = StyleSheet.create({
-  row: { flexDirection: "row", justifyContent: "space-between" },
-  euFlag: {
-    width: 84,
-    height: 56,
-    flex: 0
+  emptyHeader: {
+    flex: 1,
+    height: heightPercentageToDP("10%"),
+    backgroundColor: "red"
```

Comment:
```suggestion
```
Perhaps it remained from development phase 🔴 
<img width="300" alt="Schermata 2022-01-19 alle 18 24 38" src="https://user-images.githubusercontent.com/26501317/150182373-928e0c4d-429d-4de2-a3d1-edf13f833282.png">

Reply:
ops
https://github.com/pagopa/io-app/pull/3666/commits/985e45058254ff83faffe67e22cf73bf2bfe2301